common/djangoapps/util/disable_rate_limit.py

```diff
@@ -0,0 +1,86 @@
+"""Utilities for disabling Django Rest Framework rate limiting.
+
+This is useful for performance tests in which we need to generate
+a lot of traffic from a particular IP address.  By default,
+Django Rest Framework uses the IP address to throttle traffic
+for users who are not authenticated.
+
+To disable rate limiting:
+
+1) Decorate the Django Rest Framework APIView with `@can_disable_rate_limit`
+2) In Django's admin interface, set `DisableRateLimitConfiguration` to "enabled".
+
+Note: You should NEVER disable rate limiting in production.
+
+"""
+from functools import wraps
+import logging
+from rest_framework.views import APIView
+from util.models import DisableRateLimitConfiguration
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _check_throttles_decorator(func):
+    """Decorator for `APIView.check_throttles`.
+
+    The decorated function will first check model-based config
+    to see if rate limiting is disabled; if so, it skips
+    the throttle check.  Otherwise, it calls the original
+    function to enforce rate-limiting.
+
+    Arguments:
+        func (function): The function to decorate.
+
+    Returns:
+        The decorated function.
+
+    """
+    @wraps(func)
+    def _decorated(*args, **kwargs):
+        # Skip the throttle check entirely if we've disabled rate limiting.
+        # Otherwise, perform the checks (as usual)
+        if not DisableRateLimitConfiguration.current().enabled:
```

Comment:
No, if there isn't a record in the database, `ConfigurationModel` (the superclass) will create a new model instance and return it.